src/main/java/org/openrsc/editor/gui/menu/BrushMenu.java

```diff
@@ -0,0 +1,36 @@
+package org.openrsc.editor.gui.menu;
+
+import org.openrsc.editor.gui.GuiUtils;
+
+import javax.swing.*;
+
+public class BrushMenu extends BaseMenu {
```

Comment:
This was stubbed out on the main but not added. I replicated the stubs here using the new pattern.